bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -126,22 +126,23 @@ export default {
           equipment = findEquipment(this.element.equipment, 'Equipment_Lock', false)
           if (!equipment.length) return []
           allPoints = allEquipmentPoints(equipment)
-          points = findPoints(allPoints, 'Point', true, 'Property_Opening')
+          points = findPoints(allPoints, 'Point', true, 'Property_Opening', true)
           if (points.length) return points
           return equipment.filter((e) => e.points.length === 0).map((e) => e.item)
         case 'climate':
           equipment = findEquipment(this.element.equipment, 'Equipment_HVAC', true)
           if (!equipment.length) return []
           allPoints = allEquipmentPoints(equipment)
           points = [
-            ...findPoints(allPoints, 'Point_Status', false),
-            ...findPoints(allPoints, 'Point_Control', true)
+            ...findPoints(allPoints, 'Point_Status', false, 'Property_Power'),
+            ...findPoints(allPoints, 'Point_Control', true, 'Property_Power')
           ]
           if (points.length) return points
           return equipment.filter((e) => e.points.length === 0).map((e) => e.item)
         case 'screens':
           equipment = [
-            ...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display', true),
+            ...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display', false),
+            ...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display_Television', false),
```

Comment:
Why is this explicitly needed?
Couldn't we use 

```js
...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display', true),
```

?

Reply:
Projector is also a child of Display, and that has a separate badge. We didn’t want to break that.

Reply:
Okay, fine then :+1: